File: legacy/android/java/StorylyDemo/app/src/main/java/com/appsamurai/storylydemo/AdActivity.java
package com.appsamurai.storylydemo;

import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;
import com.appsamurai.storyly.StorylyInit;
import com.appsamurai.storyly.StorylyView;
import com.appsamurai.storyly.monetization.StorylyAdViewProvider;

public class AdActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_ad);

        StorylyView storylyView = findViewById(R.id.storyly_view);
        storylyView.setStorylyInit(new StorylyInit(Tokens.STORYLY_INSTANCE_TOKEN));
        storylyView.setStorylyAdViewProvider(new StorylyAdViewProvider(this, Tokens.ADMOB_NATIVE_AD_ID));
    }
}

File: android/kotlin/StorylyDemo/app/src/main/java/com/appsamurai/storylydemo/AdActivity.kt
package com.appsamurai.storylydemo

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.appsamurai.storyly.StorylyInit
import com.appsamurai.storyly.monetization.StorylyAdViewProvider
import com.appsamurai.storylydemo.databinding.ActivityAdBinding

class AdActivity : AppCompatActivity() {
    private lateinit var binding: ActivityAdBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityAdBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.storylyView.storylyInit = StorylyInit(STORYLY_INSTANCE_TOKEN)
        binding.storylyView.storylyAdViewProvider = StorylyAdViewProvider(this, ADMOB_NATIVE_AD_ID)
    }
}